docs: Add instructions for enabling Docker Desktop's host networking to support the Web UI. (#741)
y-scope/clp: docs/src/user-guide/quick-start-cluster-setup/single-node.md

* [Docker]
  * If you're not running as root, ensure `docker` can be run
    [without superuser privileges][docker-non-root].

>   * If you're using Docker Desktop, you will need version 4.34 or later, and [enable host networking][docker-desktop-host-networking].

Author: let me know if we want to also update the multi-node cluster requirements section. i intentionally skipped the update there because launching a multi-node cluster with Docker Desktop might be an uncommon action, unless for trial / experimental purposes.

>   * If you're using Docker Desktop, you will need version 4.34 or later, and [enable host networking][docker-desktop-host-networking].

Reviewer: ```suggestion
  * If you're using Docker Desktop, ensure version 4.34 or higher is installed, and
    [host networking is enabled][docker-desktop-host-networking].
```
